BUILD

```diff
@@ -3,10 +3,54 @@
 # SPDX-License-Identifier: Apache-2.0
 
 load("@com_github_bazelbuild_buildtools//buildifier:def.bzl", "buildifier")
+load("//rules:quality.bzl", "clang_format_check", "license_check")
 
 package(default_visibility = ["//visibility:public"])
 
 buildifier(
     name = "buildifier",
     exclude_patterns = ["./**/vendor/**"],
 )
```

Comment:
@drewmacrae should we rename this `buildifier_fix` and standardize on `*_check` and `*_fix` rules for lints?

Reply:
I like the idea of renaming it. The default behavior of buildifier is... aggressive

Reply:
Done